Add scikit-surprise

NicolasHug opened the pull request.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/scikit-surprise```) and found some lint.

Here's what I've got...


For **recipes/scikit-surprise**:

 * The recipe must have a `build/number` section.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/scikit-surprise```) and found it was in an excellent condition.

CJ-Wright commented on recipes/scikit-surprise/LICENSE.md:
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016, Nicolas Hug
```
Is the library not included in the tarball?

NicolasHug replied:
You mean the license file? Yes, it is. Should I remove it from the recipe?

CJ-Wright replied:
Yes, sorry the license. If it's in the tarball then you don't need it in the recipe

CJ-Wright commented on recipes/scikit-surprise/meta.yaml:
```diff
@@ -0,0 +1,49 @@
+
+test:
+  imports:
+    - surprise
```
Can you add some more tests than just the top level import (imports of the modules should be fine)?

NicolasHug replied:
Sure! I tried to run the full test suite by adding 
    
    -  requires:
        - pytest
    -  commands:
        - pytest

but the tests were not found.. any idea how to run tests with pytest?

I'll add other imports anyway

isuruf replied:
That's because the source is deleted at the test phase. See https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-files for how to use test files from source

NicolasHug commented:
Thanks for the feedback. I added more import tests and removed the license file. I just realized that I did not include the `tests` folder in the archive so I won't be able to use pytest here.

NicolasHug commented:
Awesome, thanks a lot for the support!
